packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -24,6 +24,16 @@ type Props = {|
      */
     onChange: (selectedValue: string) => mixed,
 
+    /**
+     * Callback for when the dropdown is opened.
+     */
+    onOpen?: () => mixed,
+
+    /**
+     * Callback for when the dropdown is closed.
+     */
+    onClose?: () => mixed,
```

Comment:
Rather than optional, what do you think about providing a default noop function?